pkg/theia/commands/get/clickhouse/command.go

```diff
@@ -0,0 +1,281 @@
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clickhouse
+
+import (
+	"database/sql"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+
+	"antrea.io/theia/pkg/theia/util"
+)
+
+type chOptions struct {
+	diskInfo    bool
+	tableInfo   bool
+	insertRate  bool
+	formatTable bool
+}
+
+type diskInfo struct {
+	shard          string
+	name           string
+	path           string
+	freeSpace      string
+	totalSpace     string
+	usedPercentage string
+}
+
+type tableInfoBasic struct {
+	shard      string
+	database   string
+	tableName  string
+	totalRows  string
+	totalBytes string
+	totalCols  string
+}
+
+type writeRowsPerSec struct {
+	shard       string
+	rowsPerSec  string
+	bytesPerSec string
+}
+
+const (
+	diskQuery = "SELECT shardNum() as shard, name as Name, path as Path, formatReadableSize(free_space) as Free," +
+		" formatReadableSize(total_space) as Total, TRUNCATE((1 - free_space/total_space) * 100, 2) as Used_Percentage FROM " +
+		"cluster('{cluster}', system.disks) ;"
+	tableInfoBasicQuery = "SELECT shard, DatabaseName, TableName, TotalRows, TotalBytes, TotalCols FROM (SELECT " +
+		"shardNum() as shard, database AS DatabaseName, name AS TableName, total_rows AS TotalRows, " +
+		"formatReadableSize(total_bytes) AS TotalBytes FROM cluster('{cluster}', system.tables) WHERE database = " +
+		"'default') as t1 INNER JOIN(SELECT shardNum() as shard, table_catalog as DatabaseName, table_name as " +
+		"TableName, COUNT(*) as TotalCols FROM cluster('{cluster}', INFORMATION_SCHEMA.COLUMNS) WHERE table_catalog == " +
+		"'default' GROUP BY table_name, table_catalog, shard) as t2 ON t1.DatabaseName = t2.DatabaseName and " +
+		"t1.TableName = t2.TableName and t1.shard = t2.shard"
+	// average writing rate for all tables per second
+	writePerSecQuery = "SELECT sd.shard, sd.Rows_per_second, sd.Bytes_per_second  FROM (SELECT shardNum() as " +
+		"shard, (intDiv(toUInt32(date_trunc('minute', toDateTime(event_time))), 2) * 2) * 1000 as t, " +
+		"TRUNCATE(avg(ProfileEvent_InsertedRows),0) as Rows_per_second, formatReadableSize(avg(ProfileEvent_InsertedBytes)) as Bytes_per_second, " +
+		"ROW_NUMBER() OVER(PARTITION BY shardNum() ORDER BY t DESC) rowNumber FROM cluster('{cluster}', " +
+		"system.metric_log) GROUP BY t, shardNum() ORDER BY t DESC, shardNum()) sd WHERE sd.rowNumber=1"
+)
+
+var options *chOptions
+
+// Command is the support bundle command implementation.
+var Command *cobra.Command
+
+var example = strings.Trim(`
+theia get clickhouse --storage
```

Comment:
Hi Yun-tang, a general comment about naming: maintainers suggest us sticking to `theia <feature> <verb> [<options>]` format.

Reply:
Thanks for reminding. I've updated.